### PR TITLE
CFY-5326 refine and standardize CLI help texts

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: cloudify_cli/config/parser_config.py [hunk 1]
```diff
@@ -34,7 +34,9 @@
 from cloudify_cli.constants import DEFAULT_INSTALL_WORKFLOW
 from cloudify_cli.constants import DEFAULT_UNINSTALL_WORKFLOW
 
-FORMAT_INPUT_AS_YAML_OR_DICT = 'formatted as YAML or as "key1=value1;key2=value2"'
+FORMAT_INPUT_AS_YAML_OR_DICT = \
+    ('Can be provided as wildcard based paths to YAML files, '
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: What are these `wildcard based paths`? What is their format, and how the user is supposed to know about it?

File: cloudify_cli/config/parser_config.py [hunk 2]
```diff
@@ -86,15 +88,15 @@ def validate_blueprint_argument():
         'dest': 'validate_blueprint',
         'action': 'store_true',
         'help': 'Validate the blueprint before uploading it '
-                'to the manager'
+                'to Cloudify Manager'
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: This is the only time that you refer to the Manager as 'Cloudify Manager' and not as 'Manager'. Why?

File: cloudify_cli/config/parser_config.py [hunk 3]
```diff
@@ -168,7 +170,7 @@ def timeout_argument(default_timeout=DEFAULT_TIMEOUT):
         'type': int,
         'default': default_timeout,
         'help': 'Operation timeout in seconds (The execution itself will keep '
-                'going. It is the CLI that will stop waiting for it to '
+                'going but the CLI will stop waiting for it to '
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Maybe a comma after 'going'?

File: cloudify_cli/config/parser_config.py (hunk 2 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: good point.

File: cloudify_cli/config/parser_config.py [hunk 4]
```diff
@@ -244,26 +246,27 @@ def auto_generate_ids_argument():
 
 def parser_config():
     return {
-        'description': 'Manages Cloudify in different Cloud Environments',
+        'description': 'A Command Line Interface Cloudify',
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: This change is great in concept. 
But what about something in the likes of "Cloudify's Command Line Interface"? I'm not familiar with the sentence structuring you proposed.

File: cloudify_cli/config/parser_config.py (hunk 3 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: yup

File: cloudify_cli/config/parser_config.py (hunk 1 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: wildcard based paths are obvious to anyone using bash... but I'll add an example in parentheses

File: cloudify_cli/config/parser_config.py (hunk 4 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: it was supposed to be '...for Cloudify' .. but I'm fine with what you're proposing

File: cloudify_cli/config/parser_config.py [hunk 5]
```diff
@@ -276,22 +279,22 @@ def parser_config():
...
                                 'dest': 'backup_first',
-                                'help': 'Whether to backup before purging.'
-                                        'Backup will be in tar.gz format.',
+                                'help': 'Whether to backup before purging. '
+                                        'Backup will be in tar.gz format',
                                 'action': 'store_true',
                             }
                         },
-                        'help': "Delete a Manager's logs",
+                        'help': "Delete the Manager's service logs",
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Is there a difference between the Manager's 'logs' and its 'service logs'? Why does the 'purge' command specifically refers to 'service logs', while the two other sub commands under logs refer to 'logs'?

File: cloudify_cli/config/parser_config.py [hunk 6]
```diff
@@ -312,18 +315,17 @@ def parser_config():
...
-                                        ' creation ({0}). '
+                        inputs_argument('Inputs for the deployment ({0}). '
                                         'This argument can be used multiple times. '
                                         '(default: {1})'
                                         .format(FORMAT_INPUT_AS_YAML_OR_DICT,
                                                 DEFAULT_INPUTS_PATH_FOR_INSTALL_COMMAND)
                                         ),
                     '-w,--workflow': make_optional(workflow_id_argument(
-                            hlp='The workflow to start '
+                            hlp='A workflow name to execute '
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: It sound a bit like we are executing the workflow's name, but I can't think of a better suggestion.

File: cloudify_cli/config/parser_config.py [hunk 7]
```diff
@@ -339,17 +341,16 @@ def parser_config():
...
                     '-d,--deployment-id': make_required(
                             deployment_id_argument(
-                                hlp='The id of the deployment you wish to '
+                                hlp='The ID of the deployment you wish to '
                                     'uninstall'
                             )
                         ),
                     '-w,--workflow': make_optional(workflow_id_argument(
-                            hlp='The workflow to start (default: {0}'
+                            hlp='A workflow name to execute (default: {0}'
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Same as above

File: cloudify_cli/config/parser_config.py (hunk 6 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: It should probably be "The name of the workflow to execute"

File: cloudify_cli/config/parser_config.py (hunk 5 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: because it was 1:30am when I did this :)

File: cloudify_cli/config/parser_config.py [hunk 8]
```diff
@@ -400,24 +401,24 @@ def parser_config():
...
                     'list': {
-                        'help': 'List all the plugins on the Manager',
+                        'help': 'List all plugins currently on the Manager',
                         'handler': cfy.plugins.ls
                     },
                     'delete': {
                         'arguments': {
                             '-p,--plugin-id': plugin_id_argument(
-                                hlp='The plugin id'),
+                                hlp="A plugin's ID"),
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Why 'A' and not 'The'?
Notice that in 'cfy snapshots delete', "--snapshot-id"'s help text is:
'The ID of the snapshot to delete'
I suggest that you decide on a format right now, and then go ahead and apply it all over this module =)

File: cloudify_cli/config/parser_config.py [hunk 9]
```diff
@@ -448,7 +449,7 @@ def parser_config():
                         'arguments': {
                             '-b,--blueprint-id': blueprint_id_argument(),
                             '-o,--output': {
-                                'help': 'The output file path of the blueprint to be downloaded',
+                                'help': 'The local path of the downloaded blueprint',
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: `cfy plugins download`'s `--output` argument states:
'A local path for the downloaded plugin',
and here `cfy blueprints download`'s `--output` states:
'**The** local path **of** the downloaded blueprint

File: cloudify_cli/config/parser_config.py [hunk 10]
```diff
@@ -244,26 +247,27 @@ def auto_generate_ids_argument():
...
+                'help': 'show version information and exit',
                 'action': cfy.version
             }
         },
         'commands': {
             'logs': {
-                'help': 'Handle Cloudify Manager logs',
+                'help': "Handle the Manager's logs",
                 'sub_commands': {
                     'get': {
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Compare `cfy blueprints download --output` and `cfy plugins download --output`, which, according to their help texts, seem to serve about the same purpose. So why here it's `logs --destination-path`?

File: cloudify_cli/config/parser_config.py (hunk 8 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: I thought I did. That's why I asked you to check for inconsistencies :) It was late at night. I'll do that right now. `The` actually makes more sense.

File: cloudify_cli/config/parser_config.py [hunk 11]
```diff
@@ -522,7 +522,7 @@ def parser_config():
                                 'dest': 'snapshot_path',
                                 'type': argparse.FileType(),
                                 'required': True,
-                                'help': "Path to the manager's snapshot file",
+                                'help': "Local path to the Manager's snapshot file",
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Add `A`\`The` as usual

File: cloudify_cli/config/parser_config.py [hunk 12]
```diff
@@ -548,75 +548,74 @@ def parser_config():
...
                             },
                             '-f,--force':
                                 force_argument(
                                         hlp='Force restoring the snapshot on '
                                             'a Manager with existing '
-                                            'blueprints and/or deployments')
+                                            'blueprints or deployments')
                         },
-                        'help': 'Restore manager state to a specific snapshot',
+                        'help': "Restore a Manager using a snapshot",
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Why did you change it to double quotes?

File: cloudify_cli/config/parser_config.py [hunk 13]
```diff
@@ -548,75 +548,74 @@ def parser_config():
...
                         'handler': cfy.snapshots.restore
                     }
                 }
             },
             'agents': {
-                'help': "Manage Cloudify's Agents",
+                'help': "Handle a deployment's agents",
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Great. Much more informative.

File: cloudify_cli/config/parser_config.py [hunk 14]
```diff
@@ -548,75 +548,74 @@ def parser_config():
...
+                'help': "Handle deployments on the Manager",
                 'sub_commands': {
                     'create': {
                         'arguments': {
                             '-d,--deployment-id': make_required(
                                 remove_completer(
                                     deployment_id_argument(
-                                        hlp='A unique id that will be assigned'
-                                            ' to the created deployment'
+                                        hlp='A unique ID to assign to the created deployment'
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: compare with `cfy snapshots create --snapshot-id`: 
`A unique ID for the snapshot`
BTW, isn't an ID unique by definition?

File: cloudify_cli/config/parser_config.py (hunk 10 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: let's forget the implementation for a second and concentrate on the help texts. If it bothers you, please open a JIRA issue for it.

File: cloudify_cli/config/parser_config.py (hunk 12 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: good q

File: cloudify_cli/config/parser_config.py [hunk 15]
```diff
@@ -668,43 +666,42 @@ def parser_config():
...
                             '--system-workflows': {
                                 'dest': 'include_system_workflows',
                                 'action': 'store_true',
-                                'help': 'Include executions of '
-                                        'system workflows'
+                                'help': 'Include executions of system workflows'
                             },
                         },
-                        'help': 'List all running executions, or all '
+                        'help': 'List all running executions on the Manager or all '
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Does `cfy execution list` list all **running** executions, or all executions, including those who already terminated?
If only running executions, then maybe change `or all executions for a specific deployment` to `or all **running** executions for a specific deployment`
If not only running executions, then delete the `running`.

File: cloudify_cli/config/parser_config.py [hunk 16]
```diff
@@ -718,42 +715,41 @@ def parser_config():
...
                         },
-                        'help': 'Cancel an execution by its id',
+                        'help': 'Cancel an execution',
                         'handler': cfy.executions.cancel
                     }
                 }
             },
             'nodes': {
-                'help': 'Manage nodes',
+                'help': "Handle a blueprint's nodes",
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: `deployment's nodes` instead of `blueprint's nodes`?

File: cloudify_cli/config/parser_config.py [hunk 17]
```diff
@@ -773,61 +769,61 @@ def parser_config():
...
                         'arguments': {
                             '--node-instance-id': {
                                 'dest': 'node_instance_id',
                                 'required': True,
-                                'help': 'The ID of the node instance to get'
+                                'help': 'The ID of the node-instance to get'
                             }
                         },
-                        'help': "Get a node instance according to its ID",
+                        'help': "Get a node instance",
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: add hyphen to `node instance`

File: cloudify_cli/config/parser_config.py (hunk 16 above)
Comment (nir0s) [inline cloudify_cli/config/parser_config.py]: oops

File: cloudify_cli/config/parser_config.py [hunk 18]
```diff
@@ -1032,28 +1024,27 @@ def parser_config():
...
-                        'help': 'If the bootstrap fails,'
-                                ' the Manager will remain running'
+                        'help': 'Do not teardown the Manager even if the bootstrap fails'
                     },
                     '--skip-validations': {
                         'dest': 'skip_validations',
                         'action': 'store_true',
-                        'help': 'Run bootstrap without '
-                                'validating resources prior to bootstrapping the manager'
+                        'help': 'Run a bootstrap without '
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Maybe without the `Run a` part. Just `Bootstrap without validating ...`.

File: cloudify_cli/config/parser_config.py [hunk 19]
```diff
@@ -1100,25 +1090,24 @@ def parser_config():
...
             'teardown': {
-                'help': 'Teardown Cloudify',
+                'help': 'Teardown the Manager',
                 'arguments': {
                     '--ignore-deployments': {
                         'dest': 'ignore_deployments',
                         'action': 'store_true',
-                        'help': 'Perform teardown even if deployments'
-                                'exist on the manager'
+                        'help': 'Perform the teardown even if there are '
```
Comment (AviaE) [inline cloudify_cli/config/parser_config.py]: Just like the previous comment, maybe start the help text with `Teardown`
